core/src/main/java/org/mapfish/print/map/tiled/CoverageTask.java

```diff
@@ -319,13 +342,6 @@ public static final class Tile {
     /** The y index of the image. the y coordinate to draw this tile is yIndex * tileSizeY */
     private final int yIndex;
 
-    /**
-     * Constructor.
-     *
-     * @param image
-     * @param xIndex
-     * @param yIndex
-     */
```

Comment:
This was not an amazing great Javadoc comment :-P

Reply:
In this case this is an internal software resource, not an external (hardware) one